go/pio/host/allwinner/allwinner.go

```diff
@@ -138,6 +139,18 @@ var Pins = []Pin{
 	{number: 113, group: 0, offset: 10, name: "PL10", defaultPull: gpio.Float},
 	{number: 114, group: 0, offset: 11, name: "PL11", defaultPull: gpio.Float},
```

Comment:
I just realized; all winner should be split in 3, one for the base registers, one separate driver for the PL registers that would only load on the appropriate CPUs then the one for r8. This will remove all the conditions from the code and will make it much more readable.

I don't think number is needed anymore, it's an error of understanding on my part.